server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -68,6 +68,8 @@ export class TenantManager implements core.ITenantManager {
             });
         };
         const defaultHeaders = getDefaultHeaders();
+        // If defaultInternalHistorianUrl is undefined or empty in the server, the tenant's internalHistorianUrl
```

Comment:
Did you mean `internalHistorianUrlOverride`?
E.g. "If internalHistorianUrlOverride is undefined or empty..."

Reply:
Yup. Done. :)